mne/io/eeglab/eeglab.py

```diff
@@ -381,6 +381,15 @@ def __init__(self, input_fname, montage, eog=(), event_id=None,
 
     def _create_event_ch(self, events, n_samples=None):
         """Create the event channel."""
+        if len(set(events[:, 0])) != len(events[:, 0]):
+            warn("Some events overlap/occur at the same time sample. The "
+                 "resulting Raw object will contain the event information "
+                 "in a stimulus channel. Stimulus channels can only carry "
+                 "one event code per time point, so some events will be "
+                 "lost. You can use the function `mne.io.read_eeglab_events`"
+                 "to extract the full events array (which can then be "
+                 "passed to e.g. `mne.Epoch`."
```

Comment:
And no reason to put backticks here, this is a warning not something Sphinx will try to render